File: client/Client/Network/Messages/Editor/EnemySpawnPointUnlockMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LoadRunnerClient.Network.Messages {
	class EnemySpawnPointUnlockMessage {

		public const string TYPE = "EnemySpawnPointUnlockMessage";

		private int _gridX, _gridY;

		public int gridX {
			get => _gridX;
			set => _gridX = value;
		}

		public int gridY {
			get => _gridY;
			set => _gridY = value;
		}

		public EnemySpawnPointUnlockMessage(int gridX, int gridY) {
			this.gridX = gridX;
			this.gridY = gridY;		
		}

		public EnemySpawnPointUnlockMessage() {
		}
	}
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/EnemySpawnPointUnlockMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.dtos.EnemySpawnPointDTO;
import de.hsrm.mi.swtpro03.server.game.dtos.ItemDTO;

public class EnemySpawnPointUnlockMessage {

	public static final String TYPE = "EnemySpawnPointUnlockMessage";

	private int gridX, gridY;
	
	public EnemySpawnPointUnlockMessage() {
	}

	public EnemySpawnPointUnlockMessage(int gridX, int gridY) {	
		this.gridX = gridX;
		this.gridY = gridY;
	}

	public int getGridX() {
		return gridX;
	}

	public void setGridX(int gridX) {
		this.gridX = gridX;
	}

	public int getGridY() {
		return gridY;
	}

	public void setGridY(int gridY) {
		this.gridY = gridY;
	}

	
	
}
